docs/server/source/production-deployment-template/troubleshoot.rst

```diff
@@ -0,0 +1,70 @@
+Cluster Troubleshooting
+=======================
+
+This page describes some basic issues we have faced while deploying and
+operating the cluster.
+
+1. MongoDB Restarts
+-------------------
+
+We define the following in the ``mongo-ss.yaml`` file:
+
+.. code:: bash
+
+    resources:
+      limits:
+        cpu: 200m
+        memory: 3.5G
+
+So, when the MongoDB cache occupies a memory greater than 3.5GB, it is
+terminated by the ``kubelet``.
+This can usually be verified by logging in to the worker node running MongoDB
+container and looking at the syslog (the ``journalctl`` command should usually
+work). It should show an error message as:
+
+TODO: paste the error message the next time this error happens!
```

Comment:
This is a placeholder till I get my hands on the precise stacktrace in kernel logs.